common/Step-ID.acn

```diff
@@ -0,0 +1,28 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Plugin was developed under a programme and funded by
```

Comment:
I've made a mistake in library-template - it's already fixed in master: it's not a Plugin, it's a Library :)